common/src/java/com/zimbra/common/soap/AdminConstants.java

```diff
@@ -1555,7 +1555,8 @@ public final class AdminConstants {
     public static final String E_NETWORK_LICENSE_CODE = "networkLicenseCode";
     public static final String E_PRODUCT_LICENSE_CODE = "productLicenseCode";
     public static final String E_CERT_ACTION_TYPE = "certActionType";
-    public static final String E_OFFLINE_LICENSE_ACTIVATION_REQUEST_CERTIFICATE = "activationRequestCertificate";
+    public static final String E_OFFLINE_NETWORK_LICENSE_ACTIVATION_REQUEST_CERTIFICATE = "networkActivationRequestCertificate";
```

Comment:
networkActivationFingerprint
productActivationFingerprint

Reply:
Renamed the field names